konig-core/src/main/java/io/konig/core/vocab/SH.java

```diff
@@ -75,6 +75,8 @@
 	public static final URI property = new URIImpl(PROPERTY);
 	public static final URI minLength = new URIImpl("http://www.w3.org/ns/shacl#minLength");
 	public static final URI maxLength = new URIImpl("http://www.w3.org/ns/shacl#maxLength");
+	public static final URI decimalPrecision = new URIImpl("http://www.w3.org/ns/shacl#decimalPrecision");
```

Comment:
We cannot make changes to the SHACL vocabulary since it is controlled by the W3C.

Please move `decimalPrecision` and `decimalScale` to the Konig vocabulary.